draft-ietf-gnap-core-protocol.md

```diff
@@ -6450,12 +6568,16 @@ to manage at runtime, and systems should generally refuse to fetch a URI if unsu
 Keys presented by value are allowed to be in only a single format,
 as discussed in {{key-format}}. Presenting the same key in multiple formats is not allowed
 and is considered an error in the request. If multiple keys formats were allowed,
-receivers of these key definitions need to be able to make sure that it's the same
+receivers of these key definitions would need to be able to make sure that it's the same
 key represented in each field and not simply use one of the key formats without checking for
 equivalence. If equivalence were not carefully checked, it is possible for an attacker to insert
 their own key into one of the formats without needing to have control over the other formats. This
 could potentially lead to a situation where one key is used by part of the system (such as
-identifying the client instance) and a different key in a different format in the same message is used for other things, like calculating signature validity.
+identifying the client instance) and a different key in a different format in the same message is
+used for other things (such as calculating signature validity). However, the primary reason to allow
+keys to be presented in multiple formats simultaneously is the assumption that the receiver cannot
```

Comment:
Formulation is a bit ambiguous : "primary reason to allow keys to be presented in multiple formats" -> actually not allowed (I understand it's for the sake of explanation, but still)